src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1139,12 +1141,12 @@
   [orig-report :- reports/report-schema
    timestamp :- pls/Timestamp
    update-latest-report? :- s/Bool]
-  (time! (:store-report metrics)
+  (time! (:store-report performance-metrics)
          (let [{:keys [puppet_version certname report_format configuration_version
                        start_time end_time resource_events transaction_uuid environment
-                       status noop]
-                :as report}         (normalize-report orig-report)
-                report-hash         (shash/report-identity-hash report)]
+                       status noop metrics logs] :as report} (normalize-report orig-report)
```

Comment:
@AJ this is more in line with your suggestion

https://gist.github.com/wkalt/712e6b672b6114f47fb4

but I'm not sure it's an improvement. I agree the destructuring in the current approach is clunky but on the other hand I think it's nice to see the full object being inserted. My preference is to leave it be, but I'm open to others' thoughts on it.